src/backend/api/reviews.js

```diff
@@ -17,7 +17,20 @@ router.get('/', async (req, res) => {
 })
 
 // Returns all reviews for a specific meal.
-
+router.get('/meal/:id', async (req, res) => {
+  try {
```

Comment:
This very much looks like a path that fetches one meal by the id. 
How would I know that this path is for reviews?